src/Krank/Checkers/IssueTracker.hs

```diff
@@ -102,31 +101,36 @@ extractIssues filePath toCheck = case parse (findAllCap patterns) filePath toChe
   Right res -> map snd $ rights res
   where
     patterns = localized $ choice [
-      githubRE
-      -- gitlabRE -- TODO: enable gitlab again
+      githubParser
+      , gitlabParser
       ]
 
 -- Supports only github for the moment
 issueUrl :: GitIssue
          -> Req.Url 'Req.Https
 issueUrl issue = case server issue of
   Github -> Req.https "api.github.com" Req./: "repos" Req./: owner issue Req./: repo issue Req./: "issues" Req./: (pack . show $ issueNum issue)
-  -- Gitlab -> Req.https "google.com"
+  Gitlab -> Req.https "gitlab.com" Req./: "api" Req./: "v4" Req./: "projects" Req./: ([fmt|{owner issue}/{repo issue}|]) Req./: "issues" Req./: (pack . show $ issueNum issue)
```

Comment:
Is the formating for owner / repo not supposed to be "encoded" somehow? https://docs.gitlab.com/ee/api/README.html#namespaced-path-encoding

Reply:
it is. that's why I pass a string and not use `Req./:`. All the strings you pass to this chain are URL encoded by `Network.HTTP.Req`

Reply:
Haaaa. Yes. Missed that fact.